classes/class-wc-connect-shipping-method.php

```diff
@@ -290,7 +290,8 @@ public function calculate_shipping( $package = array() ) {
 							)
 						),
 						'cost'     => $rate->rate,
-						'calc_tax' => 'per_item'
+						'calc_tax' => 'per_item',
+						'meta_data' => array( 'wc_connect_packages' => json_encode( $rate->packages ) ),
```

Comment:
What happens if we don't `json_encode $rate->packages`? I take it something bad happens if the key points directly at `$rate->packages`?